index.html

```diff
@@ -48,30 +51,182 @@ <h2>I'm a Software Developer</h2>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project2">
+          <h2 class="work-subtitle protitle pro2title">Cafe Bill</h2>
+          <p class="work-details prodetails">Lorem ipsum dolor sit amet consectetur adipisicing elit. Vero aliquid debitis nulla? Nulla, doloribus id consequatur.</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section class="projsecondpage">
+        <div id="project3">
+          <h2 class="work-subtitle protitle">Cat's Photo</h2>
+          <p class="work-details prodetails">Cat's Photo Project built on freecodecamp.org</p>
+          <div class="flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project4">
+          <h2 class="work-subtitle protitle pro2title">Photo Gallery</h2>
+          <p class="work-details prodetails">Cat's Photo Gallery implementation of CSS flexbox</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section class="projthirdpage">
+        <div id="project5">
+          <h2 class="work-subtitle protitle">The Quiz</h2>
+          <p class="work-details prodetails">Make a quiz templete using html forms</p>
+          <div class="flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+        <div id="project6">
+          <h2 class="work-subtitle protitle pro2title">Tribute Page</h2>
+          <p class="work-details prodetails">Tribute to Dr. Abdul Salam a Nobel Scientist.</p>
+          <div class="flex pro2flex">
+           <p class="flex-item item1 proitem1">css</p>
+           <p class="flex-item item2 proitem2">html</p>
+           <p class="flex-item item3 proitem3">bootstrap</p>
+          </div>
+        </div>
+        <button class="work-button probutton">See Project</button>
+      </section>
+      <section id="aboutme">
```

Comment:
- [ ] I would implore you to add space at this meeting point just as it is on the Figma design, by this, you will master how to adapt the Figma design.

### Your design
![image](https://user-images.githubusercontent.com/86200356/219903394-b80f3a25-5549-4750-b778-fa6434f03970.png)

### Figma Design
![image](https://user-images.githubusercontent.com/86200356/219903485-b3913a32-9dfe-4aaa-b958-11a7f316dcfb.png)

Reply:
![image](https://user-images.githubusercontent.com/112868013/219905378-e981d992-08da-4f6d-a4b3-d50ee03ec0eb.png)
@Olamarx 
it is my design